src/components/sideNav.js

```diff
@@ -132,15 +142,25 @@ export default function sideNav({
                     slug
                   }
                 }
+                Referendums {
+                  Title
+                  Description
+                  Total_Contributions
+                  fields {
+                    slug
+                  }
+                }
               }
             }
           }
         }
       `}
       render={data => {
-        const { Date, OfficeElections } = data.allElection.edges[0].node
-        const [menu, menuOptions] = formatMenu(OfficeElections)
-        const sections = Object.keys(menu)
+        const { Date, OfficeElections, Referendums } =
+          data.allElection?.edges?.[0]?.node ?? []
```

Comment:
After the recent API changes you'll need to access elections like this:

    data.allElections.edges['11/3/2020'].node

(I think I should've updated the dummy data as well)

Reply:
Got it, I'll make a new PR with all of those changes, since i'll have to jump in a ton of files.

Reply:
Actually, wait. This should be breaking everywhere else if that's the case...

Reply:
Uh yeah nevermind, this should be fine :)